network/connPerfMon_test.go

```diff
@@ -91,6 +91,7 @@ func BenchmarkConnMonitor(b *testing.B) {
 }
 
 func TestConnMonitorStageTiming(t *testing.T) {
+	t.Skip("The test ConnMonitorStageTiming is a performance benchmarking test for the various ConnectionPerformanceMonitor object stagesm and isn't a very usefull for correctness validation")
```

Comment:
mb add tag like
```
// +build: performance-test
```
and run
```
go test -tags=performance 
```
?

Reply:
build flags are for entire files, whereas we want to exclude a single function.
I could naturally move it to a separate file, but it sounds like too much trouble for custom benchmark, don't you think so ? ( especially since was written around a specific regression )